src/test/scala/com/atomist/rug/runtime/js/PlanBuilderTest.scala

```diff
@@ -0,0 +1,52 @@
+package com.atomist.rug.runtime.js
+
+import com.atomist.param.SimpleParameterValues
+import com.atomist.project.archive.{DefaultAtomistConfig, RugArchiveReader}
+import com.atomist.rug.ts.TypeScriptBuilder
+import com.atomist.source.{SimpleFileBasedArtifactSource, StringFileArtifact}
+import org.scalatest.mockito.MockitoSugar
+import org.scalatest.{FunSpec, Matchers, OneInstancePerTest}
+
+class PlanBuilderTest extends FunSpec with Matchers with OneInstancePerTest with MockitoSugar  {
+
+  val simpleCommandWithObjectInstructionParamAsJson =
+    StringFileArtifact(DefaultAtomistConfig.handlersRoot + "/Handler.ts",
+      """
```

Comment:
General point: externalizing TS into side files should make tests faster due to the potential for caching.